packages/terra-section-header/src/SectionHeader.jsx

```diff
@@ -55,6 +51,14 @@ class SectionHeader extends React.Component {
     this.state = { isActive: false };
     this.wrapOnKeyDown = this.wrapOnKeyDown.bind(this);
     this.wrapOnKeyUp = this.wrapOnKeyUp.bind(this);
+    this.setFocus = this.setFocus.bind(this);
+    this.focusableElementRef = React.createRef();
+  }
+
+  setFocus() {
```

Comment:
@cm9361  instead of having focus handler written on component we can passback the ref of section-header `this.focusableElementRef` (similar to [refCallback](https://github.com/cerner/terra-core/blob/97be1c221e0f977646be81235a1597e6ef95dd45/packages/terra-button/src/Button.jsx#L84) of button) to consumer they can call the focus() or any required native events on application end. 

So that consumer can write ` sectionHeader.current.focus();` instead of `sectionHeader.current.setFocus();`